<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>330</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>222</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>222</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>kcalc</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="button_layout"/>
   </widget>
   <widget class="QLCDNumber" name="result_display">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="numDigits">
     <number>16</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Dec</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>19</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="input_box_instructor_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>2</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input box:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>222</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="about_menu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="edit_menu">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuShow_result_in">
     <property name="title">
      <string>Show result in...</string>
     </property>
     <addaction name="actionHex"/>
     <addaction name="actionDec"/>
     <addaction name="actionOct"/>
     <addaction name="actionBin"/>
    </widget>
    <addaction name="actionCopy_the_result"/>
    <addaction name="actionSwitch_mode"/>
    <addaction name="menuShow_result_in"/>
   </widget>
   <addaction name="edit_menu"/>
   <addaction name="about_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCopy_the_result">
   <property name="text">
    <string>Copy the result</string>
   </property>
  </action>
  <action name="actionSwitch_mode">
   <property name="text">
    <string>Switch mode</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionDec">
   <property name="text">
    <string>Dec(default)</string>
   </property>
  </action>
  <action name="actionOct">
   <property name="text">
    <string>Oct</string>
   </property>
  </action>
  <action name="actionHex">
   <property name="text">
    <string>Hex</string>
   </property>
  </action>
  <action name="actionBin">
   <property name="text">
    <string>Bin</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
